config/api.php

```diff
@@ -220,4 +220,14 @@
 
     ],
 
+    'formatsOptions' => [
```

Comment:
I think this detailed styling requirements should be added to the contributing guidelines file. :)